logger/logger.go

```diff
@@ -0,0 +1,41 @@
+package logger
```

Comment:
why reimplement logger, glog can print trace, warning, and error and can adjust verbosity.

Reply:
Changed it to glog instead